string object_name
geometry_msgs/PointStamped stamped_point
bool is_selected
